book/src/developer-guides/frame-nft.md

```diff
@@ -0,0 +1,52 @@
+# Overview
+Frame-NFT is aimed at developers to provide them with a reliable and adjustable codebase for NFT-based projects. 
+This is especially necessary due to the various forms that NFTs can present as, each with their own unique traits.
+In the following we will do a quick recap of what makes NFTs special followed by how that is reflected in Frame-NFT's code.
+And finally, we will discuss two use cases.
+
+---
+
+# Recap
+NFTs (Non-Fungible-Tokens) have a few characteristics setting them apart from fungible ones like currency, precious metals, or carbon credits:
+* Non - Interchangeable 
+* Specific creator and owner
+* Unique digital signature
+* Unique asset like Art, Music or Property
+* Provide complete history of their creation and trades
+
+They represent ownership of a unique tangible or intangible asset, and record its history on the blockchain.
+
+---
+
+# Workflow
+With the nature of NFTs in mind, it becomes imperative to have a straightforward implementation aiming to accommodate their traits.
+We achieve this by having multiple methods to define distinct types and characteristics in an asset.
+Ownership being a core concept in NFTs makes the presentation and trading of ownership a vital part of how we interact with them.
+This makes storing and searching for specific information an essential part of this framework.
```

Comment:
This is a little bit vague. I'd rewrite this paragraph and instead structure it like:

## Extrinsics
*description of each extrinsic, who may call them and how they work

## Traits
* trait implementations in composable-support implemented by this pallet

## Workflow
* expected combination of extrinsics to be called to mint then transfer an NFT